parachains/chain-specs/collectives-polkadot-genesis.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "Polkadot Collectives", // TODO Collectives OR Polkadot Collectives?
+  "id": "collectives_polkadot",
+  "chainType": "Live",
+  "bootNodes": [ // TBD
+    "/ip4/127.0.0.1/tcp/30333/p2p/12D3KooWJY6kM1ES9cZV6tv7CXNcxetKVDXXmpwxzeqvTzANfDza"
+  ],
+  "telemetryEndpoints": null,
+  "protocolId": null,
+  "properties": {
+    "ss58Format": 0,
+    "tokenDecimals": 10,
+    "tokenSymbol": "DOT"
+  },
+  "relay_chain": "polkadot",
+  "para_id": 1002, // TODO review!
+  "codeSubstitutes": {},
+  "genesis": {
+    "runtime": {
+      "system": {
+        "code": "" // TODO inject when the rest is ready, otherwise this file is not reviewable on github
+      },
+      "parachainSystem": null,
+      "parachainInfo": {
+        "parachainId": 1002 // TODO review!
+      },
+      "balances": {
+        "balances": []
+      },
+      "collatorSelection": { // TODO TBD
+        "invulnerables": [
+          "5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY"
+        ],
+        "candidacyBond": 16000000000,
```

Comment:
the bond should be probably greater than 1.6 DOT

Reply:
We will probably launch without any open slots (so just the invulnerables). We can set it to whatever Statemine is.